// Generated by sylk.build, DO NOT EDIT.
syntax = "proto3";
package public.bookstore.book.v1;

import "google/protobuf/empty.proto";

service BookStoreService {
	// [public.bookstore.book.v1.BookStoreService.ListBooks] - 
	rpc ListBooks (google.protobuf.Empty) returns (stream public.bookstore.book.v1.Book);
	// [public.bookstore.book.v1.BookStoreService.GetBook] - 
	rpc GetBook (public.bookstore.book.v1.GetBookRequest) returns (public.bookstore.book.v1.Book);
}


// [public.bookstore.book.v1.GetBookRequest] - 
message GetBookRequest {
	// [public.bookstore.book.v1.GetBookRequest.Name] - 
	string Name = 1;
}

// [public.bookstore.book.v1.Book] - The bookstore main entity "Book".
message Book {
	// [public.bookstore.book.v1.Book.name] - 
	string name = 1;
}